vim-patch:8.0.{1516,1549},8.1.{44,1545}

janlazo opened the pull request.

janlazo committed:
vim-patch:8.1.1545: when the screen is to small there is no message about that

Problem:    When the screen is to small there is no message about that.
            (Daniel Hahler)
Solution:   Do not use :cquit. (closes vim/vim#4534)
https://github.com/vim/vim/commit/45aa07d3c126e887c614f8a4ebdb88aed673a9f1

blueyed commented:
Have you checked how that affects `make oldtest` in Neovim?  IIRC it is not really affected by this, is it?  (still good to have anyway, of course)

janlazo commented:
I ran `make oldtest` with different screen sizes. I can't tell the difference.

janlazo committed:
vim-patch:8.1.0044: if a test function exists Vim this may go unnoticed

Problem:    If a test function exists Vim this may go unnoticed.
Solution:   Check for a test funtion quitting Vim.  Fix tests that did exit
            Vim.
https://github.com/vim/vim/commit/8903676d3d4c3fcb37d2a55c67370f30806b1626

Restore test_assert.vim to run Test_zz_quit_detected().

janlazo committed:
vim-patch:8.0.1516: errors for job options are not very specific

Problem:    Errors for job options are not very specific.
Solution:   Add more specific error messages.
https://github.com/vim/vim/commit/b3292fa2d362b064ff8a115fc0ad794c1f1265d7

janlazo committed:
vim-patch:8.0.1549: various small problems in test files

Problem:    Various small problems in test files.
Solution:   Include small changes.
https://github.com/vim/vim/commit/5d7ead3bc85eefd0929bfcbb579510c8164ea1be
